identity_credential/Cargo.toml

```diff
@@ -59,7 +71,14 @@ validator = ["dep:itertools", "dep:serde_repr", "credential", "presentation"]
 domain-linkage = ["validator"]
 domain-linkage-fetch = ["domain-linkage", "dep:reqwest", "dep:futures"]
 sd-jwt = ["credential", "validator", "dep:sd-jwt-payload"]
-jpt-bbs-plus = ["credential", "validator", "dep:zkryptium", "dep:bls12_381_plus", "dep:json-proof-token"]
+sd-jwt-vc = ["sd-jwt", "dep:sd-jwt-payload-rework", "dep:jsonschema"]
```

Comment:
This feature also seems to need `"dep:futures"`, so:
```toml
sd-jwt-vc = ["sd-jwt", "dep:sd-jwt-payload-rework", "dep:jsonschema", "dep:futures"]
```

Without it downstream crates will encounter these compilation errors :) :
```bash
warning: `oid4vci` (lib) generated 2 warnings (run `cargo fix --lib -p oid4vci` to apply 2 suggestions)
error[E0433]: failed to resolve: use of undeclared crate or module `futures`
 --> /home/nander/Impierce/identity.rs/identity_credential/src/sd_jwt_vc/metadata/vc_type.rs:1:5
  |
1 | use futures::future::BoxFuture;
  |     ^^^^^^^ use of undeclared crate or module `futures`

error[E0433]: failed to resolve: use of undeclared crate or module `futures`
 --> /home/nander/Impierce/identity.rs/identity_credential/src/sd_jwt_vc/metadata/vc_type.rs:2:5
  |
2 | use futures::future::FutureExt;
  |     ^^^^^^^ use of undeclared crate or module `futures`

error[E0599]: no method named `boxed` found for `async` block `{async block@/home/nander/Impierce/identity.rs/identity_credential/src/sd_jwt_vc/metadata/vc_type.rs:105:3: 105:13}` in the current scope
   --> /home/nander/Impierce/identity.rs/identity_credential/src/sd_jwt_vc/metadata/vc_type.rs:153:4
    |
105 | /   async move {
106 | |     // Check if current type has already been checked.
107 | |     let is_type_already_checked = passed_types.contains(&current_type);
108 | |     if is_type_already_checked {
...   |
152 | |   }
153 | |   .boxed()
    | |   -^^^^^ method not found in `{async block@/home/nander/Impierce/identity.rs/identity_credential/src/sd_jwt_vc/metadata/vc_type.rs:105:3: 105:13}`
    | |___|
```